contents/customers/creatify.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Why Creatify switched from Google Analytics to PostHog
+customer: Creatify
+logo: ../images/customers/creatify/logo.png
+logoDark: ../images/customers/creatify/logo-dark.png
+featuredImage: ../images/customers/creatify/featured.png
+industries:
+    - AI
+users:
+    - Engineering
+    - Leadership
+    - Founder
+toolsUsed:
+    - Web analytics
+    - Product analytics
+date: 2024-03-19
+---
+
+[Creatify](https://creatify.ai/) is a powerful AI video editor that simplifies the process of quickly creating short video ads for use across any platform. Its capability to rapidly generate marketing video content allows users to test strategies and adopt the data-driven approach that Creatify champions.
+
+“We officially launched Creatify in early 2023 but we’d been in development mode for a while before that,” said founding CTO Xin Zhou, who met his Creatify co-founders while working at Meta. “We used that time to build the product and test ideas. PostHog was key to that — we’d been using it for about three months before we even launched.”
+
+Before switching to PostHog, Creatify had also tried using other platforms, such as Google Analytics and Amplitude. In both cases they found the tools unsuitable for their needs.
+
+“We started asking other founders what they recommended and they all said PostHog is better,” said Xin. “So, we tried it, and found it pretty simple to set up. Then we found out that PostHog also supports A/B testing. And feature flags. And web analytics!”
+
+## Using PostHog for web analytics
+
+When the PostHog web analytics beta first launched in November 2023, Creatify was among the first organizations to opt-in and start testing it out. As confidence grew, they quickly migrated away from Google Analytics entirely. 
+
+“Monitoring our web analytics is definitely much easier with PostHog,” said Xin. “I'm not an expert on Google Analytics and it can take me a long time to figure some things out with it. PostHog is very intuitive though. I can just immediately find the data I need, whereas in GA4 I don’t even know where to start.”
```

Comment:
```suggestion
“Monitoring our web analytics is definitely much easier with PostHog,” said Xin. “I'm not an expert on Google Analytics and it can take me a long time to figure some things out with it. PostHog is very intuitive, though. I can just immediately find the data I need, whereas in GA4 I don’t even know where to start.”
```